src/site/_collections/paged-tags.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const blogTags = require("../_data/blogTags");
+const pageContent = require("../_filters/page-content");
+const postDescending = require("./post-descending");
+
+/**
+ * Returns all posts as an array of paginated tags.
+ * It's not as if every element in the array is a single page for a tag, rather, its an array that includes every tags page.
+ * Each element includes n number of posts as well as some basic information of that tag to pump into `_includes/partials/paged.njk`
+ * This is because we can not paginate something already paginated... Pagination is effectively a loop, and we can't have an embedded loop O^2.
+ *
+ * @param {any} collection
```

Comment:
Mind filling this comment in a tad more to indicate it's an eleventy collection object.

@samthor one of the things we discussed with Ewa and Michael while you were out was starting to migrate over to typescript style comments for new code. My understanding is that since typescript is a superset of JSDoc, I don't think any of our old comments are wrong. Eventually we could add the typsecript compiler as a kind of lint step to just typecheck our code before deploying. wdyt?

Reply:
Well, some of the old comments are wrong because JSDoc is tricky and the docs are hard :)

I think it would be nice to indicate the properties/shape of this object, but I don't know if there's TypeScript definitions for it. I think ideally we should find the `.ts.d` file for a `Collection` and [reference it like this](https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html#import-types)... although this is probably awkward if we're not actually running TS, because it seems fragile to reference another file with an import like this.